letters/modernized/14-letters.adoc

```diff
@@ -225,14 +238,19 @@ considering her age, which I prize as a great mercy.
 Mine, and her dear love is to you, and your wife, and to my loving friend Henry Gouldney,
 with any other friends in your freedom.
 
+[.signed-section-closing]
```

Comment:
So, on my PDF viewer, this SSCL doesn't all fit on one line. It definitely reads like a SSCL... but when they don't fit onto one line they look kinda funny (because the second line hangs down under on the right side of the page). So, when you're looking through the chapters before sending up the PR, if you see any SSCLs that are flopping down onto two lines, there are a couple options. One is just to leave it as part of the above paragraph, and not mark it as a SSCL. The other, (which doesn't work in this particular letter) is to leave PART of it up in the above paragraph and take a CHUNK of it and make it into a SSCL. I'll give you an example of this below.